app/Filament/Pages/EditProfile.php

```diff
@@ -200,6 +226,33 @@ public function form(Form $form): Form
                     ->aside()
                     ->schema($connectedAccounts->toArray())
                     ->visible($connectedAccounts->count()),
+                Section::make('Office Hours')
+                    ->aside()
+                    ->schema([
+                        Toggle::make('office_hours_are_enabled')
+                            ->label('Enable office hours')
+                            ->live(),
+                        Toggle::make('appointments_are_restricted_to_existing_students')
+                            ->label('Restrict appointments to existing students')
+                            ->visible(fn (Get $get) => $get('office_hours_are_enabled')),
+                        Section::make('Days')
+                            ->schema($officeHoursDays)
+                            ->visible(fn (Get $get) => $get('office_hours_are_enabled')),
+                        Grid::make(3)
+                            ->schema([
+                                Toggle::make('out_of_office_is_enabled')
+                                    ->label('Out of office')
+                                    ->inline(false)
+                                    ->live()
+                                    ->visible(fn (Get $get) => $get('office_hours_are_enabled')),
+                                DateTimePicker::make('out_of_office_starts_at')
+                                    ->label('Start')
+                                    ->visible(fn (Get $get) => $get('out_of_office_is_enabled')),
+                                DateTimePicker::make('out_of_office_ends_at')
+                                    ->label('End')
+                                    ->visible(fn (Get $get) => $get('out_of_office_is_enabled')),
```

Comment:
What do you think about making this a repeater? So that someone could set multiple out-of-office events? It could be stored in a table, or even just JSON in a column is sufficient.

Reply:
I thought about that when I was working through it, but the ticket doesn't read like the intent is to have multiple out of office events at this time. I'll double check with Product, but that's how I interpreted it.

Reply:
Per Product, just one upcoming for now.